openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1395,7 +1395,14 @@ Credential Issuers publishing metadata MUST make a JSON document available at th
 
 Communication with the Credential Issuer Metadata Endpoint MUST utilize TLS.
 
-To fetch the Credential Issuer Metadata, the requester MUST send an HTTP request using the GET method and the path formed following the steps above. The Credential Issuer MUST return a JSON document compliant with this specification using the `application/json` media type and the HTTP Status Code 200.
+To fetch the Credential Issuer Metadata, the Wallet MUST send an HTTP request using the GET method and the path formed following the steps above. The Wallet is RECOMMENDED to send an `Accept-Content` Header in the HTTP GET request to indicate whether it supports signed metadata.
+
+The Credential Issuer MUST respond with HTTP Status Code 200 and return the Credential Issuer Metadata containing the [parameters defined in this specification](#credential-issuer-parameters) as either:
+
+* an unsigned JSON document using the media type `application/json`
+* a signed JSON Web Token (JWT) containing the Credential Issuer Metadata in its payload using the media type `application/jwt`
+
+The Credential Issuer MAY ignore the `Accept-Content` Header. It MUST indicate the type of the returned Credential Issuer Metadata using the HTTP `Content-Type`.
```

Comment:
I know i kinda convinced you to remove a similar sentence, but it felt weird reading without any comment on what to do.

```suggestion
The Credential Issuer MAY ignore the `Accept-Content` Header. It MUST indicate the type of the returned Credential Issuer Metadata using the HTTP `Content-Type`.
Issuer and Wallet policies regarding signed or unsigned metadata are out of scope of this document and should be defined on a ecosystem level (e.g. via a Profile).
```

Reply:
```suggestion
The Credential Issuer MAY ignore the `Accept` Header. It MUST indicate the type of the returned Credential Issuer Metadata using the HTTP `Content-Type`.
```